mesh/mesh.cpp

```diff
@@ -1486,6 +1525,34 @@ Array<int> Mesh::GetFaceToBdrElMap() const
    return face_to_be;
 }
 
+Geometry::Type Mesh::GetTypicalElementGeometry() const
+{
+   if (GetNE() > 0) { return GetElementGeometry(0); }
+
+   const int dim = Dimension();
+   if (dim == 1)
+   {
+      return Geometry::SEGMENT;
+   }
+   const int mesh_gen = MeshGenerator();
```

Comment:
Sorry for the drive by comment here, but why not just use the `meshgen` member here like elsewhere in the class? Either way I think its access and naming should be made consistent so we can ctrl-f in the library for either `meshgen` or `mesh_gen`.

Reply:
@v-dobrev I think you wrote this part. What do you think about using `meshgen` directly instead of defining the new local variable `mesh_gen`?

Reply:
I don't remember why I wrote it this way -- maybe originally I had this code outside of the `Mesh` class, so I had to access `meshgen` it with the method. I don't see a downside to using `meshgen` directly, if you prefer it that way.

Reply:
Done in f88486fda17c095408467c1e61860806c93c7d2c